### PR TITLE
Pulls/1.0/toolbar style layout

### PR DESCRIPTION
**Pull request by phalkunz**
Fixes https://github.com/silverstripe/silverstripe-admin/issues/217

### CODE REVIEW

**Comment by dhensby** [comment]
NPM tests failing :(

File: client/src/legacy/LeftAndMain.Content.js [hunk 1]
```diff
@@ -89,7 +89,9 @@ $.entwine('ss', function($){
 
       // Fix dimensions to actual extents, in preparation for a relayout via jslayout.
       this.height('auto');
-      this.height(this.innerHeight()-this.css('padding-top')-this.css('padding-bottom'));
+      const paddingTop = parseInt(this.css('padding-top'));
+      const paddingBottom = parseInt(this.css('padding-bottom'));
```
Comment (flamerohr) [inline client/src/legacy/LeftAndMain.Content.js]: I'd recommend using `parseInt(css, 10)` or `Number(css)` which will use base 10

**Comment by flamerohr** [comment]
pages with a parent seem to have a bit too much padding/margin on the left:
screen shot 2017-09-18 at 2 04 06 pm

Not sure if the this was an effect of the toolbar fixes, but see if you can fix this as well:
screen shot 2017-09-18 at 2 02 27 pm

Security has too much gap between back button and breadcrumbs
screen shot 2017-09-18 at 2 02 04 pm

**Comment by flamerohr** [comment]
Cross browser testing didn't reveal anything new with these pull requests, but it did reveal a regression from another story, which I don't think should be a blocker for this.

screen shot 2017-09-18 at 3 35 48 pm

File: client/src/styles/legacy/_actionTabs.scss [hunk 2]
```diff
@@ -215,7 +215,8 @@ $border: 1px solid darken(#D9D9D9, 15%);
 			of ss-ui-action-tabset
 		****************************************************************/
 		&.action-menus.ss-tabset {
-			margin-top: 5px;
+			margin-top: 4px;
+			float: right;
```
Comment (flamerohr) [inline client/src/styles/legacy/_actionTabs.scss]: is the float necessary?
I don't see a difference with or without it

**Pull request merged**